src/figureplotting.jl

```diff
@@ -39,7 +39,10 @@ function get_axis(fig, P, axis_kw::Dict, plot_attr, plot_args)
         # Remove arguments may not work with plotting into the scene
         delete!(plot_attr, :show_axis)
         delete!(plot_attr, :limits)
-        delete!(plot_attr, :color) # Color may contain Cycled(1), which needs the axis to get resolved to a color
+        if plot_attr.color[] isa Cycled
```

Comment:
```suggestion
        if get(plot_attr, :color, nothing) isa Cycled
```
plot_attr is a Dict therefore it should be done like that.

Reply:
This would fix the problem but I have not run the tests